### PR TITLE
Autoformat python using black

### PR DESCRIPTION
**Pull request by ianhi**
closes: https://github.com/micro-manager/pycro-manager/issues/156

### CODE REVIEW

**Comment by ianhi** [comment]
Forgot to format the notebooks. Will do tomorrow

**Comment by ianhi** [comment]
And forgot to add the sweet badge to the readme

File: pyproject.toml [hunk 1]
```diff
@@ -2,3 +2,6 @@
 testpaths = [
     "pycromanager/test",
 ]
+
+[tool.black]
+line-length = 100
```
Comment (ianhi) [inline pyproject.toml:7]: I think that lines in this repo tend to be longer than the black default 88 so i used 100. but ultimately this was a fairly arbitrary choice

Comment (henrypinkard) [reply pyproject.toml:7]: I have a wide monitor :)

**Comment by ianhi** [comment]
Ok this should be done now

**Comment by henrypinkard** [comment]
I'm digging that badge :)

**Pull request merged**